### ISSUE TITLE
Two-arch builds

### ISSUE THREAD

**Issue by mcg1969** [Anaconda-Platform/nb_conda_kernels#244]
We can't make this a true noarch package because of the `pywin32` dependency. (conda-forge can do so because they have the magical `pywin32-on-windows` package that Anaconda does not support. That may change, but not now.)

However, what Anaconda _does_ support are _virtual packages_ like `__unix` and `__win`. This would allow us to build _two_ noarch packages in parallel that cover all Python versions.

**Comment by mathause** [comment]
See https://github.com/conda-forge/nb_conda_kernels-feedstock/pull/52?

**Comment by mcg1969** [comment]
Excellent, thanks. I'll work on bringing that over here. Note that I'm thinking of this more from the perspective of testing; I want to be able to test the package in as close to its built form as possible The fact that we have noarch packages on conda-forge but we're building them arch over here is a discrepancy I don't like.

**Comment by mcg1969** [comment]
That raises another issue... I believe the scripts themselves can potentially be retired, at least on new-ish versions of notebook and/or jupyter_server.
